src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -233,6 +237,113 @@ int InvokeDelegate(host_main_fn host_main, int argc, const char** argv)
     return exit_code;
 }
 
+#if defined(PLATFORM_LINUX)
+
+std::string get_os_version()
+{
+    std::vector<std::string> qualifiers { "DISTRIB_ID=", "DISTRIB_RELEASE=" };
+
+    std::ifstream lsb_release;
+    lsb_release.open("/etc/lsb-release", std::ifstream::in);
+    if (lsb_release.is_open())
```

Comment:
Yeah, return `DISTRIB_ID` and `DISTRIB_RELEASE` (and OS Name and Version respectively). That should be fine.